isis/src/base/apps/mapmos/mapmos.cpp

```diff
@@ -0,0 +1,121 @@
+#include "mapmos.h"
+
+#include <QDebug>
+
+#include "ProcessMapMosaic.h"
+#include "PvlGroup.h"
+
+using namespace std;
+using namespace Isis;
+
+namespace Isis {
+
+  void mapmos(UserInterface &ui, Pvl *log) {
+    Cube *inCube = new Cube( ui.GetFileName("FROM"), "r");
+    mapmos(inCube, ui, log);
+  }
+
+
+ void mapmos(Cube *inCube, UserInterface &ui, Pvl *log) { 
+    // Get the user interface
+    ProcessMapMosaic m;
+
+    // Get the MatchBandBin Flag
+    m.SetBandBinMatch(ui.GetBoolean("MATCHBANDBIN"));
+
+    // Get the MatchDEM Flag
+    m.SetMatchDEM(ui.GetBoolean("MATCHDEM"));
+
+    // Get the track flag
+    bool bTrack = ui.GetBoolean("TRACK");
+    m.SetTrackFlag(bTrack);
+
+    // Gets the input file along with attributes
+    QString sInputFile = inCube->fileName();
+
+    ProcessMosaic::ImageOverlay overlay = ProcessMosaic::StringToOverlay( ui.GetString("PRIORITY") );
+
+    if (overlay == ProcessMapMosaic::UseBandPlacementCriteria) {
```

Comment:
Could we change this to from `ProcessMapMosaic` to `ProcessMosaic`. Makes it less confusing about which class has the enum.